package.json

```diff
@@ -117,9 +117,10 @@
     "request": "2.87.0",
     "reselect": "^3.0.1",
     "scrypt": "^6.0.3",
-    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git",
+    "selfkey-ui": "https://github.com/SelfKeyFoundation/selfkey-ui.git#feat/stack-1",
```

Comment:
using hashes instead of branches works great after merge, better to use them

Reply:
I know this is just for now that is why it is merging to staging/stacking. It will be changed when merging to dev